### ISSUE TITLE
Quotes in InputView are not encoded

### ISSUE THREAD

**Issue by mcsSolutions** [HomeSeer/Plugin-SDK#209]
An event action is setup using PageFactory WithInput method and default Text property.  When the Input Text Box is rendered it contains the text previously entered for this control so the user is able to edit what was previously entered.  If this text contains a quote then the remainder of the text is not shown in the Input Text Box.  This makes the text uneditable.  

Example code:
    Private Function InitNewConfigPage(ByVal iAction As Integer) As PageFactory
        Dim confPage As PageFactory = PageFactory.CreateEventActionPage(PageId, ActionName)
        confPage.WithInput(ActionInputId, "enter something")
        Return confPage
    End Function

**Comment by spudwebb** [comment]
Tracked as PSDK-210
You only see this problem with double quotes?  single quotes and other special characters are working fine, right? 

We are going to fix this bug, but as a workaround, you could use TextAreaView which works fine with quotes.

**Comment by mcsSolutions** [comment]
An example text string and the views from the two modes of the Event Action are shown below.  Typically JSON-formatted strings are used that illustrate the issue

 

zigbee2mqtt/Lamp-1/set={"brightness":255,"color":{"r":1,"g":255,"b":1}}

**Comment by mcsSolutions** [comment]
Same  behavior of everything after then first double-quote is not visible exists as well with the page factory WithTextArea as it does with WithInput.  This evaluation was done with HS 4.2 SDK.

**Comment by spudwebb** [comment]
I fixed the problem for InputView and the fix will be in next Plugin SDK release

But I can't reproduce the problem with TextAreaView. For testing I replaced WithInputView with WithTextArea in the sample plugin "Write to Log" action and it works:

![image](https://user-images.githubusercontent.com/5427056/132385098-c8633fdf-726a-4bfa-ae5f-c32c7a75c60d.png)

and here is the html source:

```
    <div id="13-449-messageinput-par" class="md-form md-outline jui-view">
        <textarea id="13-449-messageinput" class="md-textarea form-control jui-input" rows="5">Test "double quotes" : OK
Test 'simple quote' : OK</textarea>
        <label for="13-449-messageinput" id="13-449-messageinput-hint">Message</label>
    </div>
```
Can you post the generated html source code for your example that doesn't work?

**Comment by mcsSolutions** [comment]
Operator error for TextAreaView test.  Confirmed that TextAreaView is working with quotes.

**Comment by jldubz** [comment]
This was fixed in [v1.4.0.0](https://homeseer.github.io/Plugin-SDK-Docs/release_notes/pluginsdk_releasenotes_1_4_0_0.html)
